en/gps_compass/rtk_gps_locosys_m300.md

```diff
@@ -0,0 +1,101 @@
+# Locosys M300 RTK system 
+
+[LOCOSYS RTK-M300](https://www.locosystech.com/en/product/RTK-System/rtk-M300.html) is a compact and rugged advanced RTK (Real-time kinematic) receiver and industrial grade computer with Intel Atom® x5-E3930 processor dual core 1.3GHz (boost up to 1.8GHz).
+It is designed for harsh conditions, or where you need a noiseless ad-hoc network environment.
+
+![LOCOSYS RTK-M300 mini pc](../../assets/hardware/gps/locosys_m300/locosys_m300_general.png)
+
+LOCOSYS RTK-M300 supports global GPS/Glonass/Beidou/Galileo/QZSS satellites , L1+L5 Dual-frequency and multi-constellation RTK positioning solution.
+RTK-M300 provides full frequency 4G-LTE, UMTS/HSPA+ and GSM/GPRS/EDGE coverage.
+It features 10/100/1000Mbps Ethernet data and voice connectivity.
+It provides convenient SIM access via an external SIM socket
+RTK-M300 installs Win10 (or Linux) operating system, suitable for with LOCOSYS Firebird application software,provide user-friendly graphical operation interface, whether it is used for “Base station” management or “Rover” use.
+
+Whether as a RTK Base station or RTK Rover, due to its fan-less and compact design it is eaasy to install and use.
+In particular, it is suitable for base stations where space is at a premium.
+RTK-M300 keeps the flexibility to meet different demands for telemetric monitor or surveying applications.

```

Comment:
This is a bit of a mess - you are trying to say too much. I gave it a subedit but it still needs more work.